lightbug_http/header.mojo

```diff
@@ -11,6 +11,7 @@ struct HeaderKey:
     alias CONTENT_TYPE = "content-type"
     alias CONTENT_LENGTH = "content-length"
     alias CONTENT_ENCODING = "content-encoding"
+    alias TRANSFER_ENCODING = "transfer-encoding"
```

Comment:
I assume you just added it for now so we have it? I think we're not processing the transfer encoding yet?